Sources/SwiftFormat/Rules/BeginDocumentationCommentWithOneLineSummary.swift

```diff
@@ -11,6 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
+import NaturalLanguage
```

Comment:
Please add `#if os(macOS)` around this import to match where it's used, since it won't exist on Linux.

Reply:
Oh, I missed it.